### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by hm90121**
A brief description of the changes in this PR:

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- 0chain:
- Other: ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ For most transactions, `zwallet` uses the `0dns` to discover the network nodes,
 
 **Prerequisites**
 
-- go 1.13
+- Go: Installation instructions for Mac, LInux and Windows can be found here.
```
Comment (Kishan-Dhakan) [inline README.md]: `here` should be a link right?

**Review by Kishan-Dhakan — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (hm90121) [reply README.md]: actually this point has to be removed the installation guides contains instructions for installing go.

now i have made a commit to update build instructions and remove this

File: README.md [hunk 2]
```diff
@@ -1434,12 +1390,12 @@ Response:
...
-0chain-core-sdk [INFO]   2021/05/17 21:23:13.116083 wallet.go:453: https://five.devnet-0chain.net/miner01/v1/client/put
-0chain-core-sdk [INFO]   2021/05/17 21:23:13.116289 wallet.go:453: https://five.devnet-0chain.net/miner03/v1/client/put
-0chain-core-sdk [INFO]   2021/05/17 21:23:13.117002 wallet.go:453: https://five.devnet-0chain.net/miner02/v1/client/put
-0chain-core-sdk [DEBUG]  2021/05/17 21:23:13.120009 wallet.go:475: https://five.devnet-0chain.net/miner01/v1/client/put200 OK
-0chain-core-sdk [DEBUG]  2021/05/17 21:23:13.139268 wallet.go:475: https://five.devnet-0chain.net/miner03/v1/client/put200 OK
-0chain-core-sdk [DEBUG]  2021/05/17 21:23:13.139654 wallet.go:475: https://five.devnet-0chain.net/miner02/v1/client/put200 OK
+0chain-core-sdk [INFO]   2021/05/17 21:23:13.116083 wallet.go:453: https://demo.zus.network/miner01/v1/client/put
+0chain-core-sdk [INFO]   2021/05/17 21:23:13.116289 wallet.go:453: https://demo.zus.network/miner03/v1/client/put
+0chain-core-sdk [INFO]   2021/05/17 21:23:13.117002 wallet.go:453: https:///demo.zus.network/miner02/v1/client/put
```
Comment (peterlimg) [inline README.md]: There's an extra `/`, before demo, i.e `https:///demo.zus.network`

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hm90121) [reply README.md]: removed extra /

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by hm90121** [comment]
added all the changes in https://github.com/0chain/zwalletcli/pull/244
closing this

**Pull request closed without merging**